doc/Overview/Markup/Overview.md

```diff
@@ -0,0 +1,169 @@
+                .HorizontalAlignment(HorizontalAlignment.Center)
+                .Children
+                (
+                    new TextBlock()
+                        .Text("Hello Uno Platform!")
+                )
+        );
+    }
+}
+```
+
+The first thing you will notice with C# Markup is that there is nothing special to learn. You can simply create a new instance of the controls you want to work with (eg `new TextBlock()`) and set properties using the generated extension method with the same name (eg `.Text("Hello Uno Platform!')`). 
+
+Unlike XAML which is littered with string constants, C# Markup provides a strongly typed API for setting properties. For example, in XAML you would set `HorizontalAlignment` with `HorizontalAlignment="Center"`, but in C# Markup you would use the `HorizontalAlignment` enum with `HorizontalAlignment.Center`, ensuring that you get compile time validation and intellisense for all the available values.
+
+## Getting Started
+
+Let's take a look at how to get started with C# Markup. We'll start with a simple sample that displays a counter and a button that increments the counter by a step size. The sample will have a ViewModel that has a `Count` and a `Step` property, as well as a `IncrementCommand` that increments the `Count` by the `Step` when executed. 
+
+### Constructor and Properties
+
+We'll start with creating a control and setting properties, as shown in the following example that creates a `TextBlock`. The fluent API allows you to chain together multiple properties, as shown in the following example that sets the `Margin`, `TextAlignment` and `Text` properties.
+
+```cs
+new TextBlock()
+    .Margin(12)
+    .TextAlignment(TextAlignment.Center)
+    .Text("Counter: 0")
+```
+
+Similar to the `HorizontalAlignment` property mentioned earlier, the `TextAlignment` property is set using an enum rather than a string constant. This ensures that you get compile time validation and intellisense for all the available values.
+
+In XAML you would can set the `Margin` property using a single number (for example `<TextBlock Margin="12" />`), but the actual `Margin` property on the `TextBlock` class requires a `Thickness` type, resulting in the following C# Markup:
+
+```cs
+new TextBlock().Margin(new Thickness(12))
+```
+However, C# Markup provides automatic type conversion for common types such as `Thickness`, as well as a number of other types such as `Brush`, `Color`, `CornerRadius`, `FontFamily`, `Geometry` and `ImageSource`. This means that you can set the `Margin` property using a single number, as shown in the following example.
+
+```cs
+new TextBlock().Margin(12)
+```
+
+
+### Data Binding
+
+Since our counter example requires the value of the `TextBlock` to be updated each time the counter changes, we'll need to use data binding. C# Markup provides a strongly typed API for data binding, as shown in the following example that binds the `Text` property to the `Count` property of the ViewModel.
```

Comment:
```suggestion
### Data Binding

Since our counter-example requires the value of the `TextBlock` to be updated each time the counter changes, we'll need to use data binding. C# Markup provides a strongly typed API for data binding, as shown in the following example that binds the `Text` property to the `Count` property of the ViewModel.
```

Reply:
It's not a counter-example, it's an example counter app :-)